README.md

```diff
@@ -1,24 +1,201 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
 
-Things you may want to cover:
+  <br/>
+  <h1>Motor's Reservation</h1>
 
-* Ruby version
+</div>
 
-* System dependencies
+## 📗 Table of Contents
 
-* Configuration
+- [📖 About the Project](#about-project)
+  - [Schema](#ER-diagram)
+  - [🛠 Built With ](#-built-with-)
+  - [Tech Stack ](#tech-stack-)
+  - [Key Features ](#key-features-)
+  - [🚀 Live Demo](#live-demo)
+  - [Kanban Board](#kanban)
+  - [Video Presentation](#video-presentation)
+- [💻 Getting Started ](#-getting-started-)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Run Tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+- [🔭 Future Features:](#-future-features)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
-* Database creation
+<!-- PROJECT DESCRIPTION -->
 
-* Database initialization
+# Motor's Reservation <a name="about-project"></a>
 
-* How to run the test suite
+> The Backend for the Motor's Reservation app. Built using Ruby on Rails.
 
-* Services (job queues, cache servers, search engines, etc.)
+<!-- ER diagram"-->
 
-* Deployment instructions
+# Database Schema <a name="ER-diagram"></a>
 
-* ...
+<div align="center">
+ <img src="https://github.com/binodbhusal/final_capstone_backend/blob/dev/app/ER_diagram/drawSQL-motorhub-final-capstone-project-export-2023-09-29.png" alt="schema" width="600"  height="auto" />
+  <br/>
+</div>
+
+#### Link to [Frontend](https://github.com/binodbhusal/final_capstone_frontend)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Framework</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Authentication
+- CRUD
+- Image Upload
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
Replace key features like this : 
1. User Authentication: The system allows users to create accounts and  log in
2. Motors Listings: Motors providers can list their available motors. Each listing includes details such as the brand, model, manufacturer, price, description, and location.
3. Reservation Management: Users can view available motors at different locations and make reservations
4. Add and delete motors in including location

Reply:
I think the CRUD give full details